src/components/miscellaneous/TopbarNotifications.tsx

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState, useMemo } from 'react';
+import parseISO from 'date-fns/parseISO';
+import { Link } from 'react-router-dom';
+import { Notification } from 'types';
+import { useNotifications, useUpdateNotification } from 'hooks/Notification';
+import { getTimeSince, isExternalURL } from 'utils';
+
+// Material-ui
+import { Collapse, Skeleton, List, ListItem, ListItemText, ListItemIcon, Divider, Typography, IconButton, styled } from '@mui/material';
+
+// Icons
+import NotificationUnreadIcon from '@mui/icons-material/NotificationsRounded';
+import NotificationReadIcon from '@mui/icons-material/NotificationsNoneRounded';
+import LinkIcon from '@mui/icons-material/LinkRounded';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMoreRounded';
+import ExpandLessIcon from '@mui/icons-material/ExpandLessRounded';
+
+// Project components
+import NotFoundIndicator from 'components/miscellaneous/NotFoundIndicator';
+import Linkify from 'components/miscellaneous/Linkify';
+import Pagination from 'components/layout/Pagination';
+import Dialog from 'components/layout/Dialog';
+import { useGoogleAnalytics } from 'hooks/Utils';
+import Paper from 'components/layout/Paper';
+
+const NotifcationsDialog = styled(Dialog)({
+  '& .MuiPaper-root': {
+    maxWidth: '600px',
+  },
+});
+
+type NotificationItemProps = {
+  notification: Notification;
+};
+
+type NotificationTopBarProps = {
```

Comment:
Eksporter gjerne type til props på komponenter som eksporteres fra en fil

```suggestion
export type NotificationTopBarProps = {
```